src/OrangePaymentSlipData.php

```diff
@@ -130,19 +133,19 @@ public function getWithBankingCustomerId()
      * @param string $referenceNumber The reference number.
      * @return $this The current instance for a fluent interface.
      * @throws DisabledDataException If the data is disabled.
-     * @throws PaymentSlipException If the length exceeds 26 when adding banking customer length.
+     * @throws PaymentSlipException If the length exceeds REFNR_MAX_LENGTH when adding banking customer length.
      */
     public function setReferenceNumber($referenceNumber)
     {
         if (!$this->getWithReferenceNumber()) {
             throw new DisabledDataException('reference number');
         }
-                
-        if (strlen($referenceNumber) + $this->bankingCustomerIdLength > 26) {
+
+        if (strlen($referenceNumber) + $this->bankingCustomerIdLength > $this::REFNR_MAX_LENGTH) {
```

Comment:
Why do you use ``this`` instead of ``static``?
http://php.net/manual/en/language.oop5.late-static-bindings.php